packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -605,3 +622,17 @@ function updateHandleCursor(handles: HTMLElement[], angleRad: number) {
         handle.style.cursor = `${rotateHandles(handle, angleRad)}-resize`;
     });
 }
+
+/**
+ * Check if the image must be responsive
+ * @param img The current image.
+ * @param imgInfo the current edit state of the image
+ */
+function setImageResponsive(img: HTMLImageElement, wasResized: boolean) {
+    if (!wasResized) {
+        img.style.maxWidth = '100%';
+        img.style.height = 'initial';
```

Comment:
as I mentioned, you need to set all four values. For css values, you can set to initial, or even null. For HTML attributes, just delete them from the image.